src/osd/ECBackend.cc

```diff
@@ -500,6 +500,8 @@ void ECBackend::continue_recovery_op(
 	assert(!op.recovery_progress.first);
 	dout(10) << __func__ << ": canceling recovery op for obj " << op.hoid
 		 << dendl;
+	// XXX: Doing this on the read error handling just loops in recovery
+	// How would this case ever actually happen?
```

Comment:
K.  It probably wouldn't hurt to add an explanatory comment if you want to.  I had to spend 20 minutes trying to figure out why it didn't get handled by check_recovery_sources...